src/App.tsx

```diff
@@ -15,71 +17,98 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string;
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
```

Comment:
```suggestion
type Props = {};
export class App extends React.Component<Props, State> {
```
It's better to write type `Props` even when it's empty